downstream/modules/platform/con-hs-eda-controller.adoc

```diff
@@ -7,9 +7,11 @@ With {EDAcontroller}, you can set up horizontal scaling for your events automati
 The following node types are used in this deployment:
 
 API node type:: Responds to the HTTP REST API of {EDAcontroller}.
-Worker node type:: Runs an {EDAName} worker, which is the component of Event-Driven Ansible that not only manages projects and activations, but also executes the activations themselves.
+Worker node type:: Runs an {EDAName} worker, which is the component of {EDAName} that not only manages projects and activations, but also executes the activations themselves.
 Hybrid node type:: Is a combination of the API node and the worker node.
 
+// This content is used in RPM installation
+ifdef::aap-install[]
 The following example shows how you would set up an inventory file for horizontal scaling of {EDAcontroller} on {RHEL} VMs using the host group name `[automationedacontroller]` and the node type variable `eda_node_type`:
```

Comment:
Not a deal breaker by any means, but I think we are encouraged to avoid conditionals. You could sub in something like "can"?

Reply:
I didn't write the original content (I was more focused on the ifdef conditionals) but I'll update to reflect the feedback 👍